src/tests/java/htsjdk/samtools/ValidateSamFileTest.java

```diff
@@ -411,7 +411,7 @@ public void testIndexFileValidation() throws Exception {
         final File outFile = File.createTempFile("validation", ".txt");
         outFile.deleteOnExit();
         final PrintWriter out = new PrintWriter(outFile);
-        new SamFileValidator(out, 8000).setValidateIndex(true).validateSamFileSummary(samReader, reference);
+        new SamFileValidator(out, 8000).setIndexValidationStringency(BamIndexValidator.IndexValidationStringency.EXHAUSTIVE).validateSamFileSummary(samReader, reference);
         final LineNumberReader reader = new LineNumberReader(new FileReader(outFile));
```

Comment:
Could you please also add a test here (or even add it to this existing test) to test the lessExhaustivelyTestIndex() method?